perun-ldapc/src/main/resources/log4j.properties

```diff
@@ -1,39 +0,0 @@
-# log4j Configuration
```

Comment:
How is this replaced in a logback version ? I believe, own file was used on purpose - e.g. to make ``cz.metacentrum.perun.core`` log to ``perun-ldapc.log``. For now, start stop scripts do not pass logging config on command line.